packages/styles/components/Table.module.scss

```diff
@@ -23,6 +23,7 @@
 
     &:first-of-type {
       border-top-left-radius: var(--border-radius);
+      text-align: start;
```

Comment:
Just a fun fact, `start` is used taking into account the direction of browser (LTR or RTL), so it has worked in us context but will be align on right in browsers that uses RTL 😂

https://developer.mozilla.org/pt-BR/docs/Web/CSS/text-align